test/test_l2_projection.jl

```diff
@@ -121,7 +120,12 @@ function test_projection_mixedgrid()
 
     quadset = 1:1
     triaset = 2:3
-    mesh = Grid(cells, nodes)
+    return Grid(cells, nodes), quadset, triaset
```

Comment:
Should we add a test where we have a third cell set which is excluded from the L2-projection? It seems like it should work, but would be nice test.